Airports/EGNJ/SMR/Geo.txt

```diff
@@ -295,235 +286,198 @@ N053.34.46.658 W000.20.49.947 N053.34.46.834 W000.20.49.883 smrYellow
+N053.33.57.243 W000.21.24.014 N053.33.57.164 W000.21.24.090 smrYellow
+N053.33.57.164 W000.21.24.090 N053.33.57.064 W000.21.24.295 smrYellow
+N053.33.57.064 W000.21.24.295 N053.33.57.019 W000.21.24.453 smrYellow
+N053.33.57.019 W000.21.24.453 N053.33.56.982 W000.21.24.622 smrYellow
+N053.33.56.982 W000.21.24.622 N053.33.56.964 W000.21.24.801 smrYellow
+N053.33.56.964 W000.21.24.801 N053.33.56.952 W000.21.25.005 smrYellow
+N053.33.56.952 W000.21.25.005 N053.33.56.952 W000.21.25.174 smrYellow
+N053.33.56.952 W000.21.25.174 N053.33.56.937 W000.21.25.450 smrYellow
+N053.33.56.937 W000.21.25.450 N053.33.56.885 W000.21.25.675 smrYellow
+N053.33.56.885 W000.21.25.675 N053.33.56.815 W000.21.25.879 smrYellow
+N053.33.56.815 W000.21.25.879 N053.33.56.727 W000.21.26.074 smrYellow
+N053.33.56.727 W000.21.26.074 N053.33.56.612 W000.21.26.232 smrYellow
+N053.33.56.612 W000.21.26.232 N053.33.56.496 W000.21.26.348 smrYellow
+N053.33.56.496 W000.21.26.348 N053.33.56.363 W000.21.26.407 smrYellow
+N053.33.56.363 W000.21.26.407 N053.33.56.226 W000.21.26.454 smrYellow
+N053.33.56.226 W000.21.26.454 N053.33.56.079 W000.21.26.448 smrYellow
+N053.33.56.079 W000.21.26.448 N053.33.55.918 W000.21.26.395 smrYellow
+N053.33.55.918 W000.21.26.395 N053.33.55.802 W000.21.26.307 smrYellow
+N053.33.55.802 W000.21.26.307 N053.33.55.690 W000.21.26.165 smrYellow
+N053.33.55.690 W000.21.26.165 N053.33.55.573 W000.21.26.005 smrYellow
+N053.33.55.573 W000.21.26.005 N053.33.55.514 W000.21.25.876 smrYellow
+N053.33.55.514 W000.21.25.876 N053.33.55.452 W000.21.25.698 smrYellow
+N053.33.55.452 W000.21.25.698 N053.33.55.412 W000.21.25.531 smrYellow
+N053.33.55.412 W000.21.25.531 N053.33.55.374 W000.21.25.317 smrYellow
+N053.33.55.374 W000.21.25.317 N053.33.55.357 W000.21.25.145 smrYellow
+N053.33.55.357 W000.21.25.145 N053.33.55.374 W000.21.24.946 smrYellow
+N053.33.55.374 W000.21.24.946 N053.33.55.405 W000.21.24.713 smrYellow
+N053.33.55.405 W000.21.24.713 N053.33.55.475 W000.21.24.504 smrYellow
+N053.33.55.475 W000.21.24.504 N053.33.55.573 W000.21.24.288 smrYellow
+N053.33.55.573 W000.21.24.288 N053.33.57.529 W000.21.22.894 smrYellow
+N053.33.57.529 W000.21.22.894 N053.33.57.704 W000.21.22.829 smrYellow
+N053.33.57.704 W000.21.22.829 N053.33.57.822 W000.21.22.809 smrYellow
+N053.33.57.822 W000.21.22.809 N053.33.57.971 W000.21.22.829 smrYellow
+N053.33.57.971 W000.21.22.829 N053.33.58.116 W000.21.22.866 smrYellow
+N053.33.58.116 W000.21.22.866 N053.33.58.228 W000.21.22.887 smrYellow
+N053.33.58.228 W000.21.22.887 N053.33.58.361 W000.21.22.912 smrYellow
+N053.33.58.361 W000.21.22.912 N053.33.58.513 W000.21.22.900 smrYellow
+N053.33.58.513 W000.21.22.900 N053.33.58.684 W000.21.22.871 smrYellow
+N053.33.58.684 W000.21.22.871 N053.33.58.863 W000.21.22.755 smrYellow
+N053.33.58.863 W000.21.22.755 N053.33.59.115 W000.21.22.586 smrYellow
+;Holds
+N053.34.54.496 W000.20.49.492 N053.34.54.275 W000.20.49.973 smrOrange ;B
+N053.34.51.523 W000.20.51.816 N053.34.51.069 W000.20.52.769 smrOrange ;S
```

Comment:
You seem to have two 'S's, should one of them be 'A'?? And alphanumberic order once that's sorted please.

Reply:
Will need fixing in the KMZ too I think.